src/common/Formatter.cc

```diff
@@ -117,8 +117,10 @@ void Formatter::dump_format_unquoted(const char *name, const char *fmt, ...)
 
 // -----------------------
 
-JSONFormatter::JSONFormatter(bool p)
-: m_pretty(p), m_is_pending_string(false)
+JSONFormatter::JSONFormatter(bool pretty, bool format)
```

Comment:
Nit: can you just add a setter method (i.e. ```enable_line_break()```) -- I'd like to avoid creating new defaulted parameters.